proto/path/gateway.proto

```diff
@@ -59,6 +61,12 @@ message GatewayObservations {
 
   // parallel_request_observations tracks the outcome of parallel requests within a batch.
   optional GatewayParallelRequestObservations gateway_parallel_request_observations = 8;
+
+  // fallback_used indicates whether a fallback URL was used for the request.
+  bool fallback_used = 9;
```

Comment:
Similarly for other fallback* fields: shannon.proto (or a new file under proto/path/protocol/) seems like a  better fit.